amber/amber-settings.php

```diff
@@ -306,6 +321,33 @@ public function print_delivery_section_info()
     /** 
      * Get the settings option array and print one of its values
      */
+    public function amber_post_types_callback()
+    {
+        $amber_post_types_args = array(
+           'public'   => true
+        );        
+        $output = 'objects';
+        $operator = 'and';
+        $post_types = get_post_types( $args, $output, $operator ); 
```

Comment:
It looks like ```$amber_post_types_args``` isn't being used - is that supposed to be the same as ```$args```?

Also, I think it's fine to pass the strings 'objects' and 'and' as parameters to ```get_post_types()``` - no need to create separate variables.